### ISSUE TITLE
为什么self.src_node_updater.update（）和self.dst_node_updater.update（）里面更新的东西不一样呢？

### ISSUE THREAD

**Issue by MyGithub1234567890** [PengLinzhi/DyGKT#4]
您好，还有个问题 请问一下 关于        src_nodes_features = src_nodes_neighbor_node_raw_features + src_nodes_edge_raw_features + src_nodes_neighbor_time_features #+ src_nodes_neighbor_struct_features  # torch.cat((src_nodes_neighbor_node_raw_features, src_nodes_edge_raw_features),dim=-1) 
        dst_nodes_features = dst_nodes_neighbor_node_raw_features + dst_nodes_edge_raw_features + dst_nodes_neighbor_time_features #+ dst_nodes_neighbor_struct_features  # torch.cat((dst_nodes_neighbor_node_raw_features, dst_nodes_edge_raw_features), dim=-1) 

        src_node_embeddings = self.src_node_updater.update(
            src_nodes_features[:, :-1, :] + src_nodes_neighbor_skill_struct_features+ src_nodes_neighbor_struct_features) + (src_nodes_edge_raw_features + src_nodes_neighbor_time_features)[:,-1, :]
        
        if self.ablation in ['q_qid', 'q_kid']:
            dst_node_embeddings = dst_nodes_neighbor_node_raw_features[:,-1]
        else:
            dst_node_embeddings = self.dst_node_updater.update((dst_nodes_edge_raw_features + dst_nodes_neighbor_time_features)[:, :-1, :]+ dst_nodes_neighbor_struct_features) + dst_nodes_features[:,-1,:]为什么self.src_node_updater.update（）和self.dst_node_updater.update（）里面更新的东西不一样呢？

**Comment by PengLinzhi** [comment]
在这个任务中，由于学生id分布分散，如同one-hot embedding不存在语义信息，学生（dst_neighbor）id对于题目dst是一个干扰，不需要添加到dst的更新中，重要的是获取其时间、结构和连边本身的特征这三个信息。而对于某个学生来说，做过的题目id号是集中有效的节点特征，因此是需要额外添加上的。
